heron_wsgi/admin_lib/i2b2metadata.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, metadatasm, i2b2meta_schema):
         self._mdsm = metadatasm
         self.i2b2meta_schema = i2b2meta_schema
 
-    def project_terms(self, i2b2meta_schema, i2b2_pid, rc_pids,

```

Comment:
note the call to `project_terms` from `_integration_test` below.

mypy would catch this for us if we used static type annotations